ios/App/App.xcodeproj/project.pbxproj

```diff
@@ -364,9 +363,11 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = App/App.entitlements;
-				CODE_SIGN_STYLE = Automatic;
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
+				CODE_SIGN_STYLE = Manual;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = QAJW2K5S55;
+				DEVELOPMENT_TEAM = "";
+				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = PZYM8XX95Q;
```

Comment:
We don't need this SDK specific setting. I think that's something Xcode adds when doing changes via the "Signing & Capabilities" panel.

Also, moving these settings to an `xcconfig` would make changes clearer as well as bypassing Xcode making these unnecessary edits.